include/fty_common_db_asset.h

```diff
@@ -310,6 +310,12 @@ namespace DBAssets {
     std::string
     get_status_from_db (tntdb::Connection conn,
                         std::string &element_name);
+
+// select_daisy_chain: get daisy-chain of which element_id is part based on
+// daisy_chain ext properties, or empty map if not part of a daisy-chain
+// (1 -> assetId1, 2 -> assetId2...)
+    db_reply <std::map <int, uint32_t> >
+    select_daisy_chain (tntdb::Connection &conn, uint32_t element_id);
```

Comment:
I suggest to use string internal name (likes edpu-22) than integer primery key (likes 22)